### DISCUSSION TITLE
Auto fill in Maui for Android

### DISCUSSION THREAD

**Discussion by EdwardMcFlurry** [CommunityToolkit/Maui#1072] [Q&A]
Hello

I have both the Email and Password on my Login page that I’m requiring from my Users. On Android, the Email is already remembered on the keyboard and that’s where I was wondering if there’s a solution for accessing Password Manager to auto fill the User’s Credentials. On iOS, it works like a charm that it automatically prompts the User to auto fill Credentials from Passwords.

Any help/assistance will be highly appreciated.

**Comment by EdwardMcFlurry** [comment]
Anyone with any suggestions?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
